packages/endpoints-organizations-registrar/src/controllers/consents/controller.js

```diff
@@ -57,23 +56,20 @@ const consentsController = async (fastify) => {
           user,
         } = req;
 
-        const newConsent = {
-          userId: user.sub,
-          version,
-          consentId: nanoid(),
-          type,
-        };
+        const organization =
```

Comment:
this is so ugly in js. I so wish that they had option types. Its almost worth creating a higher order function to handle it
```
const organization = optional(() => await repos.organizations.findOneByDid(did, {_id: 1,}, [did]);

Reply:
`optional` should not handle errors agree? just replace null values with the last argument? Also specifically null, or any falsey values. Wdyt @sloops77 ?

Reply:
implemented in [fa4094b](https://github.com/velocitycareerlabs/velocitycore/pull/34/commits/fa4094ba70c664c581207522c8c0ad1f1dd4f9a2), wdyt @sloops77